Searching/Linear/linear.js

```diff
@@ -0,0 +1,27 @@
+const readline = require('readline')
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+var numbers = [];
+var target = 0;
+
+rl.question('Input arrays: ', (num) => {
+    rl.question('Input target: ', (tar) => {
+        numbers = num.split(" ").map(Number)
+        target = tar;
+        search(numbers, target);
+        rl.close();
+    });
+});
+
+const search = (arr, target) => {
+    for(i in arr) {
+        console.log(`Index ${i} = ${arr[i]}`);
+        if (i === target) {
+            console.log(`## Found target in index ${arr[i]}`);
```

Comment:
Suppose to be `[i]`, @ichromanrd ?

Reply:
Updated in commit https://github.com/engineers-indonesia/algorithms/pull/4/commits/d18ee2fa40c88bc26389e14686d2ffb37abb9c05 bis wkwkw